codice/ddf
# DDF-3663 adds ability to export list of metacards to a KMZ file

#### What does this PR do?
* adds action provider for KML and KMZ
* adds ability to export single metacards as KMZ
* adds ability to export multiple metacards as KMZ

#### Who is reviewing it? 
@bellcc 
@bdeining 
@derekwilhelm 
@Bdthomson 
@Schachte 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining 
@coyotesqrl
@djblue 

#### How should this be tested?
* Build / install
* Ingest multiple metacards with location data
* Verify appearance of KML/KMZ export options on individual metacard export dropdown menu
* Verify appearance of KML/KMZ export options on multiple metacard export dropdown menu
* Verify ability to export single metacard as KMZ. Export the same metacard as KML. Unzip the exported KMZ and verify its contents within `doc.kml` match those of the exported KML doc.
* Verify ability to export multiple metacards as KMZ. Export the same metacards as KML. Unzip the exported KMZ and verify its contents within `doc.kml` match those of the exported KML doc.

#### Any background context you want to provide?

#### What are the relevant tickets?
For Jira:
[DDF-3663](https://codice.atlassian.net/browse/DDF-3663)

#### Screenshots

#### Checklist:
- [ X ] Update / Add Unit Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
